apps/classroomio-com/src/lib/MoreFeatures/MoreFeatures.svelte

```diff
@@ -36,6 +36,31 @@
     }
   ];
 
+  const tools = [
+    {
+      src: '/free-tools/progress-report.svg',
+      title: 'Progress Report',
+      subText:
+        'Generate cool reports & monitor progress in real-time. Share reports with your network for collaborative learning',
+      show: true
+    },
+
+    {
+      src: '/free-tools/activity-stopwatch.svg',
+      title: 'Activity Timer',
+      subText:
+        'Stay on track and enhance productivity with our customizable timer for timed tasks, quizzes, and study sessions',
+      show: true
+    },
+    {
+      src: '/free-tools/tic-tac.svg',
+      title: 'Tic tac toe game',
+      subText:
+        "More than just a game; it's an educational tool that teaches pattern recognition, and decision-making.",
+      show: true
+    }
+  ];
```

Comment:
If you noticed you are making the classic Don't Repeat Yourself (DRY) mistake. You know there is a place on the `/tools` route where we have an array of tools and we render it on the UI. You are also duplicating the exact same content here.

To solve this in `src/lib`, create a folder called `constants` then create a file called `tools.ts`. Export an array with all our tools as something like:


```ts
export const TOOLS = [
 // All tools here
]
```